pkg/segment/segment.go

```diff
@@ -20,7 +20,9 @@ import (
 )
 
 // Writekey will be the API key used to send data to the correct source on Segment
-var WriteKey = "R1Z79HadJIrphLoeONZy5uqOjusljSwN"
+// odo-prod R1Z79HadJIrphLoeONZy5uqOjusljSwN
+// odo-dev 4xGV1HV7K2FtUWaoAozSBD7SNCBCJ65U
+var writeKey = "R1Z79HadJIrphLoeONZy5uqOjusljSwN"
```

Comment:
Out of curiosity - this key is OK to be even put out in a public code base like odo's. Why are we changing it to a private variable in that case?

Reply:
>this key is OK to be even put out in a public code base like odo's.

Yes, we confirmed this with a few people. We need those keys to be able to send data to Segment, hence they will be in our user's system. I understand you had a concern about us receiving borked data with these keys out in the open, but that's a risk we are willing to take it seems.

Reply:
@valaparthvi the question in my review is - "Why are we changing it to a private variable in that case?" I understand that it's fine to put that key in public.

Reply:
My bad. Anytime I read something about the key being open, my fingers tend to type those words :facepalm:

Reply:
> @valaparthvi the question in my review is - "Why are we changing it to a private variable in that case?" I understand that it's fine to put that key in public.

Good practise, there is no need for this variable to be public, and it should not be used outside of the `segment` package